packages/strapi-parts/src/TextInput/TextInput.stories.mdx

```diff
@@ -11,11 +12,32 @@ import { Tooltip } from '../Tooltip';
 
 # TextInput
 
-This is the doc of the `TextInput` component
+TextInputs are inputs that allow users to add or edit text. They support several text formats including numbers.
 
-## TextInput
+**Best practices**
 
-Description...
+- Icons are always on the right side within the input (writing area) itself, never on the left side.
+- TextButton component should always be above the input. 
+- Labels should be clear and concise. A few words only and without punctuation.
+- Use description line as a help to give more details about the required action.
+- Within a form, unless users need a lot of space to write their content, the input should take 1 column space instead of two.
```

Comment:
TextInput is always part of a form (the first part of the sentence is not necessary)